src/msw/listctrl.cpp

```diff
@@ -875,6 +879,25 @@ bool wxListCtrl::GetItem(wxListItem& info) const
     return success;
 }
 
+// Check if the item is visible
+bool wxListCtrl::IsVisible(long item) const
+{
+    bool result = ::SendMessage( GetHwnd(), LVM_ISITEMVISIBLE, (WPARAM) item, 0 );
+    if( result )
+    {
+        HWND hwndHeader = (HWND) ::SendMessage( GetHwnd(), LVM_GETHEADER, 0, 0 );
```

Comment:
ListView_GetHeader() macro can be used. It is used multiple times in this file.

Reply:
@MaartenBent ,
I think its a matter of preference.
To me the macro represent an old MFC code, but wx talks native Win32 API.

Reply:
@MaartenBent ,
Do you still want this change?

Reply:
It has nothing to do with MFC, it's  a win32 macro to simplify getting the header: https://docs.microsoft.com/en-us/windows/win32/api/commctrl/nf-commctrl-listview_getheader

It is already used in multiple places in this file, you could use it here as well.

Reply:
@MaartenBent ,
It does not - it just a mtter of preferences.
As I said "FOR ME" or IMHO, the macro represents an old MFC code.

But I will make the change.